libraries/SITL/SITL.cpp

```diff
@@ -234,6 +234,8 @@ const AP_Param::GroupInfo SITL::var_info3[] = {
     AP_GROUPINFO("THML_SCENARI",  12, SITL,  thermal_scenario, 0),
 
     AP_GROUPINFO("GPS2_HDG",      13, SITL,  gps_hdg_enabled[1], 0),
+    // Buyoancy for submarines
+    AP_GROUPINFO_FRAME("BUOYANCY", 14, SITL, buoyancy, 1, AP_PARAM_FRAME_SUB),
```

Comment:
I think this is your problem.  Duplicate parameter numbers.

Reply:
oops.. thanks!

Reply:
fixed btw! I probably introduced it while rebasing.